fs2/src/main/scala/laserdisc/fs2/RedisClient.scala

```diff
@@ -105,10 +99,7 @@ object RedisClient {
       ): F[Out]
     }
 
-    def mkClient[F[_]: Effect: LogWriter](connection: F[Connection[F]])(
-        implicit ec: ExecutionContext,
-        scheduler: Scheduler
-    ): F[(RedisClient[F], F[Unit])] =
+    def mkClient[F[_]: ConcurrentEffect: Timer: LogWriter](connection: F[Connection[F]]): F[(RedisClient[F], F[Unit])] =
```

Comment:
I think scalafmt is changing this to the version above.

Reply:
doesn't look like it
```
sbt:laserdisc> fs2/scalafmtCheck
[success] Total time: 2 s, completed 28-Sep-2018 17:01:30
```